client/reader/recommendations/controller.js

```diff
@@ -36,5 +42,36 @@ export default {
 
 		trackPageLoad( basePath, fullAnalyticsPageTitle, mcKey );
 		titleActions.setTitle( i18n.translate( 'Recommended Sites For You ‹ Reader' ) );
+	},
+
+	// Post Recommendations - Used by the Data team to test recommendation algorithms
+	recommendedPosts( context ) {
+		var WarmstartStream = require( 'reader/recommendations/posts' ),
+			basePath = route.sectionify( context.path ),
+			fullAnalyticsPageTitle = ANALYTICS_PAGE_TITLE + ' > Warmstart',
```

Comment:
warm start is an internal name, i don't think we really want it in stats or variable names